taichi/program/compile_config.cpp

```diff
@@ -2,7 +2,7 @@
 
 TLANG_NAMESPACE_BEGIN
 
-const bool advanced_optimization = true;
+bool advanced_optimization = true;
```

Comment:
Why not move this inside the `ti.cfg`?

Reply:
This is just a temporary solution to disable advanced optimization. Moving this to `ti.cfg` needs too many changes.